README.md

```diff
@@ -168,7 +172,8 @@ Or, if you use Maven:
 
 ## Relevant Links
 * [skUnity forums](https://forums.skunity.com)
-* [Add-on releases at skUnity](https://forums.skunity.com/forums/addon-releases)
+* [skUnity Add-on releases](https://forums.skunity.com/forums/addon-releases)
+* [skUnity Discord invite](https://discord.gg/skript)
```

Comment:
```suggestion
* [skUnity Discord invite](https://discord.gg/0l3WlzBPKX7WNjkf)
```
The invite link you mentioned doesn't have the most stable track record

Reply:
But `.gg/skript` is the custom one which is better and more rememberable

Reply:
I talked to Kenzie about it, skUnity had lost the URL before so I don't think we should rely on it